library/spdm_common_lib/libspdm_com_context_data.c

```diff
@@ -467,6 +467,12 @@ libspdm_return_t libspdm_set_data(void *context, libspdm_data_type_t data_type,
         }
         spdm_context->local_context.is_requester = *(bool *)data;
         break;
+    case LIBSPDM_DATA_REQUEST_RETRY_TIMES:
+        if (data_size != sizeof(uint8_t)) {
+            return LIBSPDM_STATUS_INVALID_PARAMETER;
+        }
+        spdm_context->retry_times = *(uint8_t *)data;
+        break;
```

Comment:
Though it is requester-only, I think we don't need to add a `is_requester` check here.
1. if it is checked, maybe a mechanism should be introduced along with it, that libspdm or integrator to guarantee the set data sequence (such as, set is_requester first, then set retry times only for requester), it would make things more complicated IMO.
2. Responder never uses retry times (so far), no harm.

Reply:
Yeah that's fine. 

As an aside, we don't impose any ordering on `set_data`, such that it would check `is_requester`. The expected flow is the Integrator populates the SPDM context and once that is complete they can call `libspdm_check_context` to check that all the different parameters are legal together.